docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -9,13 +9,15 @@ signs transaction with a key stored in an encrypted file or an external vault (f
 * `ethsigner [Options] file-based-signer [File Options]`
 * `ethsigner [Options] hashicorp-signer [Hashicorp Options]`
 * `ethsigner [Options] azure-signer [Azure Options]`
+* `ethsigner [Options] multifile-based-signer [Multifile Options]`
```

Comment:
See comment below - probably makes sense to group the two file based options together

Reply:
Done.